cargo-marker/src/lints.rs

```diff
@@ -6,22 +6,51 @@ use std::{
 
 use crate::ExitStatus;
 
+pub struct LintCrateSpec<'a> {
+    package_name: Option<&'a str>,
+    dir: &'a Path,
+}
+
+impl<'a> LintCrateSpec<'a> {
+    pub fn new(package_name: Option<&'a str>, dir: &'a Path) -> Self {
+        Self { package_name, dir }
+    }
+
+    /// Currently only checks for semicolons, can be extended in the future
+    pub fn validate(&self) -> bool {
```

Comment:
I think the polarity of the return is confusing. I was surprised, that a later check `crate_entries.iter().any(LintCrateSpec::validate)` errored if any call returns true. I would have expected `true` to indicate that everything has been validated. Maybe `is_valid` would be a better name.

Reply:
Oh, yeah, you're right, that totally needs to be renamed :sweat_smile: